packages/labs/gen-manifest/src/index.ts

```diff
@@ -57,7 +58,10 @@ const convertModule = (module: Module): cem.Module => {
     description: 'TODO', // TODO
     declarations: [...module.declarations.map(convertDeclaration)],
     exports: [
-      // TODO
+      ...module.exportNames.map((name) =>
+        convertJavascriptExport(name, module.getExportReference(name))
+      ),
+      ...module.getCustomElementExports().map(convertCustomElementExport),
```

Comment:
See https://github.com/webcomponents/custom-elements-manifest/blob/main/schema.d.ts#L83-L156 for details... basically exports contains `js` exports (actual module exports), as well as `custom-element-definition` exports, which are CE's registered in the global registry by tag name.